plover/steno.py

```diff
@@ -21,6 +21,78 @@
 _NUMBERS = set('0123456789')
 _IMPLICIT_NUMBER_RX = re.compile('(^|[1-4])([6-9])')
 
+def compare_normalize_stroke(x, y):
+    for i in range(len(x.strokes)):
+        if i > len(y.strokes) - 1:
+            return 1
+        x_norm = normalize_stroke(x.strokes[i])
```

Comment:
Since the function is "compare_normalize" I think you don't need to normalize the stroke before compare, it's implied.